docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -4,25 +4,20 @@ By [Phillip Carter](https://github.com/cartermp)
 
 **Some details are subject to change as the toolchain evolves.**
 
-Targeting .NET Core for libraries can be done entirely with the .NET CLI tools, which are a foundational set of tools used by Visual Studio, ASP.NET, and other technologies.  They provide an efficient and low-level experience that works across any supported OS.  You can still build libraries with Visual Studio, and if that is your preferred experience then you should [refer to the Visual Studio guide](libraries-with-vs.md).  This document will focus on using the CLI tools directly.
+Targeting .NET Core for libraries can be done entirely with the .NET CLI tools, which are a foundational set of tools used by Visual Studio and ASP.NET Core.  They provide an efficient and low-level experience that works across any supported OS.  You can still build libraries with Visual Studio, and if that is your preferred experience then you should [refer to the Visual Studio guide](libraries-with-vs.md).  This document will focus on using the CLI tools directly.
 
 * [Prerequisities](#prerequisites)
 * [How to target .NET Core](#how-to-target-net-core)
 * [How to target .NET Framework](#how-to-target-net-framework)
 * [How to target a Portable Class Library (PCL)](#how to target a portable class-library-pcl)
 * [How to cross-compile for .NET Core and .NET Framework](#how-to-cross-compile-for-net-core-and-net-framework)
+* [How to use multiple projects](#how-to-use-multiple-projects)
 * [How to test libraries on .NET Core](#how-to-test-libraries-on-net-core)
 * [How to create a NuGet Package with your Library](#how-to-create-a-nuget-package-with-your-library)
 
 ## Prerequisites
 
-You must have .NET Core installed on your machine.  You have two options:
-
-* [DNX](http://aspnet.readthedocs.org/en/latest/getting-started/index.html), which is part of ASP.NET 5 RC1 and is included in Visual Studio Update 1.
-
-* [.NET CLI](http://dotnet.github.io/getting-started/), which will be used by .NET Core and ASP.NET RC2 and will replace DNX.
-
-Either toolchain can be used to build and publish libraries today.
+You must have .NET Core installed on your machine.  You will need [the .NET Core SDK and CLI](http://dotnet.github.io/getting-started/).
 
 The sections of this document dealing with .NET Framework versions or Portable Class Libraries (PCLs) need the .NET Framework installed.  This is only supported on Windows.
 
```

Comment:
You start this section by saying you need .NET Core installed and then here you have this lose sentence about installing the .NET Framework. This probably belongs in the last paragraph.